cfgov/cfgov/settings/base.py

```diff
@@ -152,6 +152,11 @@
 
 ALLOWED_HOSTS = ['*']
 
+#Enabling compression for use in base.html
+COMPRESS_ENABLED = True
```

Comment:
Yep, working on a fix to allow caching but turning off the compression. Also agree that it would be better to handle the bundle generation Webpack, if possible.